examples/react/json-based-microapp-renderer-with-css/src/components/Microapp.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright 2022 Workgrid Software
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useState } from 'react'
+import { AdaptiveCardUsingHostConfigContext } from 'adaptivecards-react'
+import { useQuery, gql } from '@apollo/client'
+import Loader from './Loader'
+
+// GraphQL Query for microapp content
+const GET_MICROAPP_QUERY = gql`
+  query GetMicroapp($spaceId: ID!, $entrypoint: String!, $data: JSONObject) {
+    me {
+      space(spaceId: $spaceId) {
+        appView(entrypoint: $entrypoint, data: $data) {
+          view
+        }
+      }
+    }
+  }
+`
+
+const Microapp = ({ node }) => {
+  const [entrypoint, setEntrypoint] = useState(node.entrypoint)
+  const [queryData, setQueryData] = useState(undefined)
```

Comment:
Are these actually two independent values, or should they always be set together?

Reply:
They should always be set together - updating this.

Reply:
Updated - what do you think of the omit approach? A bit strange feeling.